src/helpers/checkName.js

```diff
@@ -3,7 +3,7 @@ import { snakeCase } from 'lodash'
 const checkName = (name) => {
   let newName = name
   // check name matches regex
-  const regex = /^[a-zA-Z0-9_][a-zA-Z0-9_.-]*[a-zA-Z0-9_]$/
+  const regex = /^[a-zA-Z_][a-zA-Z_]*[a-zA-Z_]$/
```

Comment:
```suggestion
  const regex = /^[a-zA-Z0-9_]+$/
```

- Allows digits
- Allows asset names of one or more characters - yours required two.